### ISSUE TITLE
What does softsel mean？

### ISSUE THREAD

**Issue by guotong1988** [allenai/bi-att-flow#62]
`softsel` appears many times..
Thank you..

**Comment by cmjxll** [comment]
It looks like computing attended query vector. logits is the correlation coefficient, target is document vector or query vector. You will look more clearly in bi_attention() function about in 487 rows of model.py.
